docs/snippets/common/storybook-test-runner-hooks-example.js.mdx

```diff
@@ -0,0 +1,24 @@
+```js
+// .storybook/test-runner.js
+
+module.exports = {
+  // Hook to execute once the test runner is ready
+  setup() {
+    // Add your configuration here.
```

Comment:
it's documented on the docs page itself (where this snippet is rendered)

Reply:
The preRender and postRender methods too, but the comments in this file still explain them 🤔

Reply:
@yannbf in my latest commit I've polished it a bit to "Hook that is executed before the test runner starts running tests" so that follows what's documented and unifies the documentation. Are you good with it? Let me know and we'll go from there.

Reply:
sounds great!